config/initializers/decidim_menu.rb

```diff
@@ -0,0 +1,51 @@
+Decidim.menu :menu do |menu|
```

Comment:
It was noted in another discussion that the instance is hosting also other organizations, so this should be somehow made organization specific.

One approach that I could come up with would be to define a `:metadecidim_menu` here and then customize `Decidim::MenuHelper` to render different menus for different organizations.

If you can come up with some other solutions, I am also open for alternatives.

Reply:
> If you can come up with some other solutions, I am also open for alternatives.

I couldn't come up with another one, neither. It's done in c89664bb156ce1c31db1c77030a1754b8868d406